lua/weapons/arccw_base/sh_firing.lua

```diff
@@ -728,7 +728,7 @@ function SWEP:DoRecoil()
 
         self.RecoilAmount     = self.RecoilAmount + (self.Recoil * rmul)
         self.RecoilAmountSide = self.RecoilAmountSide + (self.RecoilSide * irec * recs * rmul)
-        self.RecoilPunchBack  = math.Clamp(self.Recoil * recv * math.Rand(2, 3), 0, 2)
+        self.RecoilPunchBack  = math.Clamp(self.Recoil * recv * math.Rand(2, 3), 0, 2 * recv)
```

Comment:
this allows VisualRecoilMult to actually have a noticeable effect, instead of being inadvertently clamped to nothing